sdcm/sct_events/database.py

```diff
@@ -101,6 +102,8 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                                    regex="integrity check failed")
 DatabaseLogEvent.add_subevent_type("BOOT", severity=Severity.NORMAL,
                                    regex="Starting Scylla Server")
+DatabaseLogEvent.add_subevent_type("STOP", severity=Severity.WARNING,
```

Comment:
Why is it warning?

Reply:
I set it as warning just because "stopping scylla" must be followed by "start scylla" operation.
If you think it is not needed then I don't mind and can change it to "normal" type of event.

Reply:
let's make it a normal severity

Reply:
Done